ci_framework/roles/build_containers/defaults/main.yml

```diff
@@ -31,6 +31,10 @@ cifmw_build_containers_openstack_release: master
 cifmw_build_containers_dist_major_version: "{{ ansible_distribution_major_version }}"
 cifmw_build_containers_registry_namespace: podified-main-centos9
 cifmw_build_containers_push_registry: quay.io
+cifmw_build_containers_images: []
+cifmw_build_containers_rpms_paths: []
+cifmw_build_containers_tags: []
+cifmw_build_containers_run_build: false
```

Comment:
I would add `_hotfix` prefix to all hotfix related vars. Like this.
```
cifmw_build_containers_hotfix_images: []
cifmw_build_containers_hotfix_rpms_paths: []
cifmw_build_containers_hotfix_tag: []
```
I would replace `cifmw_build_containers_run_build` with `cifmw_build_containers_run_hotfix`.
and `cifmw_build_containers_hotfix_tag` will be string.

Reply:
yes,  _hotfix is the correct prefix